pkg/storage/cacher_test.go

```diff
@@ -236,7 +237,7 @@ func TestWatch(t *testing.T) {
 	}
 	defer initialWatcher.Stop()
 
-	verifyWatchEvent(t, initialWatcher, watch.Added, podFoo)
+	t.Log("verifying")
```

Comment:
Is it really "1" in those places?
Etcd itself is doing some initial work, so I never get the first operation to get the index of 1. It's usually 3, so substracting 1 shouldn't cause any problems.